app/views/content.js

```diff
@@ -0,0 +1,40 @@
+import Mn from 'backbone.marionette';
+import HeaderView from './header';
+import TracksView from './tracks';
+import application from '../application';
+
+
+const Content = Mn.LayoutView.extend({
+    
+    template: require('./templates/content'),
+
+    regions: {
+        header: '[role="header"]',
+        tracks: '[role="tracks"]',
+    },
+
+    onBeforeShow: function() {
+        this.showChildView('header', new HeaderView());
+        this.switchPlaylist(application.allTracks);
```

Comment:
Probably better to pass an event to call a switch, which'll be catched by the component itself